files.go

```diff
@@ -57,8 +57,9 @@ func newFilesClusterCommon(cluster Cluster) Files {
 			AbsolutePath: "/etc/kubernetes/ssl/calico/client-key.pem",
 			Data:         cluster.CalicoClient.Key,
 		},
-		// Etcd client.
-		// TODO create separate etcd client certificates.
+		// Temporary Etcd client keys reusing server keys.
+		// TODO Remove these when operator support for new flannel & calico
+		// specific etcd client keys has been rolled out.
```

Comment:
Maybe it would be good to put current operator versions here, so we know when we are safe to delete it. WDYT?

Reply:
Interesting idea but I do not follow how you want this to be. How should this look like?

Reply:
Like

```
Version bundle versions at the time of writing this comment:

    - flannel-operator: v...
    - azure-operator: v0.1.0
    - ...
```

Then we know when those versions bundles (and all lower versions) are removed we are safe to remove migration code.

Reply:
Versioning in comments will not work.

Reply:
How can you know when you can remove this code otherwise?

Reply:
It is ok now Pawel. Do what you think is right. Disagree and commit. I do not have strong feelings here and no further motivation to get deeper into the topic right now because my focus is actually somewhere else and I want to keep it like this. Did not want to stress this here.

Reply:
It isn't my PR :D 

I'm also fine with not stating current versions. But I just wonder how to delete such (at some point) dead code.